rust/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+# We name the package `synapse` so that things like logging have the right
+# logging target.
+name = "synapse"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.61.0"
```

Comment:
Is this the version you decided on? I thought I saw talk of 1.58 in synase-dev but might be confused

Reply:
Yeah, I think 1.61.0 is a reasonable one. It's from May (so four months old) and is what the latest firefox requires.

Its mostly pulled out of thin air though.